nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceNode.java

```diff
@@ -53,13 +53,15 @@
 
 import java.lang.reflect.InvocationTargetException;
 import java.net.URL;
-import java.util.ArrayList;
+import java.util.Map;
```

Comment:
can you keep alphabetical order here? weird that checkstyle is passing nonetheless...